CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+<a name="1.1.0"></a>
```

Comment:
Please make this closer to the format used by the old JS repo's [CHANGELOG](https://raw.githubusercontent.com/Shopify/javascript/f10bf7ddbdae07370cfe7c94617c450257731552/CHANGELOG.md).  i.e.,:

- Don't use more than one newline to break up content
- Don't embed anchors (Github already adds an anchor for every heading; you just have to mouse hover to see them)
- All release headings are `##` levels deep
- Break down changes into `Added`/`Changed`/`Fixed` sections
- ... `###` levels deep
- Link to PRs (e.g. `/pulls/12345` instead of `commit/abc012`) because PRs can encompass multiple commits
- Use backticks to highlight technical terms (e.g., `allow-null-check` instead of 'allow-null-check')